docs/DEVGUIDE.md

```diff
@@ -18,6 +28,22 @@ layout:
     ├── pkg            # Contains all non-"main" Go packages
     └── vendor         # Glide-managed dependencies (untracked)
 
+## Working on Issues
+
+Github does not allow non-maintainers to assign, or be assigned to, issues.
```

Comment:
Not sure - but I did find this: https://github.com/kubernetes/contrib/issues/1161  and asked a follow-on question...